docs/v3/api-ref/settings-ref.mdx

```diff
@@ -218,7 +218,7 @@ This configuration settings option specifies the path to an SSL certificate file
 
 **Type**: `string | None`
 
-**Default**: `/usr/lib/ssl/cert.pem`
+**Default**: `None`
```

Comment:
It's weird that this changed. I think it's because the `default` for this pulls from the environment. Maybe we should make that a `defaultFactory`. Definitely not blocking for this PR though.

Reply:
i think this is appropriately changed in a recent PR [here](https://github.com/PrefectHQ/prefect/pull/18801/files#r2308037606) bc @chrisguidry might have had a weird version of something related to the pre-commit/push hooks